src/consensus/params.cpp

```diff
@@ -372,16 +376,10 @@ namespace Consensus {
 
         // Funding streams are disabled if Canopy is not active.
         if (NetworkUpgradeActive(nHeight, Consensus::UPGRADE_CANOPY)) {
```

Comment:
This isn't strictly necessary; `GetActiveFundingStreams(nHeight)` will be empty if Canopy is not enabled.